rasa_nlu/server.py

```diff
@@ -44,6 +44,8 @@ def create_argparser():
     parser.add_argument('--pipeline',
                         help="The pipeline to use. Either a pipeline template "
                              "name or a list of components separated by comma")
+    parser.add_argument('--server_model_dirs',
+                        help="address of server_model_directory")
```

Comment:
Please use the description from https://rasahq.github.io/rasa_nlu/0.9.2/config.html#server-model-dirs (or something more descriptive than the current sentence :D)

Reply:
Done :)